### PR TITLE
First part was implemented

### PR DESCRIPTION
**Pull request by Mandybur800**

### CODE REVIEW

**Review by Artificial-Friend — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+
+            } catch (NumberFormatException e) {
```
Comment (Artificial-Friend) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:86]: Try to write any single number and press `Enter`

**Review by Artificial-Friend — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+                    continue;
+                }
+                bet = new Bet(value, risk);
+
+            } catch (NumberFormatException e) {
+                System.out.println("Incorrect input, please enter again");
+            } catch (ArrayIndexOutOfBoundsException e) {
+                System.out.println("You need to input two values!");
+            }
+            betDao.add(bet);
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: let's add bet only if it's not null

File: src/main/java/core/basesyntax/model/Game.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
+package core.basesyntax.model;
+
+public class Game {
+    private String sport;
+    private String team1;
+    private String team2;
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/model/Game.java]: don't user numbers in variables naming

**Review by dmytro-hryhoruk — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java (hunk 2 above)
Comment (Mandybur800) [reply src/main/java/core/basesyntax/controller/ConsoleHandler.java]: done

File: src/main/java/core/basesyntax/model/Game.java (hunk 3 above)
Comment (Mandybur800) [reply src/main/java/core/basesyntax/model/Game.java]: done

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request merged**